### PR TITLE
Render ADF previews through Glamour

### PR DESCRIPTION
**Pull request by seflue**
Builds on #74 — now that adf-converter is the editor's roundtrip path,
this hooks its display module up to Glamour so the read-side preview
uses the same node coverage.

Opt-in:

```yaml
renderer: glamour
```

Default (and `builtin`) keeps the hand-rolled renderer. The Glamour
path covers everything the converter already accepts.

Two things adf-converter/display handles on top of plain Glamour
because Markdown can't express them:

- `textColor` marks render as ANSI-colored runs (via a custom goldmark
  node)
- `underline` marks render as ANSI underline

Bumped `adf-converter` and `adf-converter/display` to v0.2.0, which
integrates two additional fixes affecting `converter: adf-converter`:
blockquote wrappers now terminate with a blank line, and inline nodes
survive the taskList roundtrip intact.

### CODE REVIEW

File: pkg/tui/views/adf.go [hunk 1]
```diff
@@ -11,16 +12,36 @@ import (
...
+	adfconv "github.com/seflue/adf-converter/adf"
+	adfdisplay "github.com/seflue/adf-converter/display"
+
 	"github.com/textfuel/lazyjira/v2/pkg/tui/components"
 	"github.com/textfuel/lazyjira/v2/pkg/tui/theme"
 )
 
+// activeRenderer selects the ADF renderer. Set once at startup from config.
+// Empty string and "builtin" both resolve to the built-in renderer.
+var activeRenderer string
```
Comment (textfuel) [inline pkg/tui/views/adf.go]: Maybe move renderer selection to an interface (like ADFConverter)? Two impls (builtin, glamour) injected via constructor, no package global.
Would also unblock a couple of small tests for both render paths since we have none right now. WDYT?

File: pkg/tui/views/adf.go [hunk 2]
```diff
@@ -36,6 +57,36 @@ func renderADF(node any, width int) []string {
...
+	}
+	var doc adfconv.Document
+	if err := json.Unmarshal(raw, &doc); err != nil {
+		return []string{fmt.Sprintf("[glamour: unmarshal: %v]", err)}
+	}
+	if width < 10 {
+		width = 10
+	}
+	out, err := adfdisplay.Render(&doc,
+		adfdisplay.WithStyle("dark"),
```
Comment (textfuel) [inline pkg/tui/views/adf.go]: On a light terminal this will be unreadable. As a minimum we could pick dark or light via `lipgloss.HasDarkBackground`.
Better would be a `rendererStyle` config option (dark, light, auto, notty) so users can override when autodetect fails (`tmux`, `ssh`)

**Review by textfuel — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks clean overall. Left a couple of comments inline

File: pkg/tui/views/adf.go (hunk 1 above)
Comment (seflue) [reply pkg/tui/views/adf.go]: I introduced an ADFRenderer interface as you suggested and also some tests.

File: pkg/tui/views/adf.go (hunk 2 above)
Comment (seflue) [reply pkg/tui/views/adf.go]: Implemented. `adf-converter` already supports this, I just forgot to use this in
lazyjira.

**Review by textfuel — APPROVED** [review]
I was busy last month, thanks for your patience.

LGTM!
